toast-tk-dao/src/main/java/io/toast/tk/dao/service/dao/access/repository/RepositoryDaoService.java

```diff
@@ -23,20 +23,20 @@
 import com.google.inject.name.Named;
 import com.mongodb.WriteConcern;
 
+import io.toast.tk.dao.domain.impl.common.IServiceFactory;
 import io.toast.tk.dao.domain.impl.repository.RepositoryImpl;
 import io.toast.tk.dao.service.dao.common.AbstractMongoDaoService;
 import io.toast.tk.dao.service.dao.common.CommonMongoDaoService;
 import io.toast.tk.dao.service.init.DbStarter;
 
 public class RepositoryDaoService extends AbstractMongoDaoService<RepositoryImpl> {
 
-	public interface Factory {
-
-		RepositoryDaoService create(final @Nullable @Assisted String dbName);
+	public interface Factory extends IServiceFactory<RepositoryDaoService>{
 	}
 
 	private static final Logger LOG = LogManager.getLogger(RepositoryDaoService.class);
 
+	@Deprecated
 	static final String CONTAINER_TYPE = "swing page";

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AS1133)